spec/system/kudo_spec.rb

```diff
@@ -7,26 +7,15 @@
     driven_by(:rack_test)
   end
 
-  let(:employee2) { build(:employee, email: 'test2@test.com') }
-  let(:employee3) { build(:employee, email: 'test3@test.com') }
+  let!(:employee) { create(:employee) }
 
-  it 'Setup 2 employees and crud kudo' do
+  it 'crud kudo' do
     visit root_path
-    click_link 'Sign Up'
-    fill_in 'Email', with: employee2.email
-    fill_in 'Password', with: employee2.password
-    fill_in 'Password confirmation', with: employee2.password
-    click_button 'Sign up'
-    expect(page).to have_content 'Welcome! You have signed up successfully.'
-    click_link 'Log Out'
-    expect(page).to have_content 'Log in'
-
-    click_link 'Sign Up'
-    fill_in 'Email', with: employee3.email
-    fill_in 'Password', with: employee3.password
-    fill_in 'Password confirmation', with: employee3.password
-    click_button 'Sign up'
-    expect(page).to have_content 'Welcome! You have signed up successfully.'
+    click_link 'Log In'
```

Comment:
FYI - there is a helper, for that:
https://sajadtorkamani.com/how-to-login-in-capybara-tests-when-using-devise/